safehttp/incoming_request.go

```diff
@@ -41,6 +41,21 @@ func newIncomingRequest(req *http.Request) *IncomingRequest {
 	}
 }
 
+// GetMethod TODO
```

Comment:
Add the comments to functions you are implementing in this PR. Otherwise I don't know what I'm reviewing.